docs/examples/docker-registry-secret.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: quarks.cloudfoundry.org/v1alpha1
+kind: QuarksSecret
+metadata:
+  name: generate-docker-registry-secret
+spec:
+  type: dockerconfigjson
+  secretName: gen-docker-registry-secret
+  imageCredentials:
+    registry: fake-registry
+    username: fake-username
+    password: fake-password
```

Comment:
storing password value in a CR is not safe because contents are not encrypted (unlike contents of Secret kind).

additionally it seems that this conflates generation functionality of QuarksSecret with encoding of existing data into a different format (no generation going on).